spec/models/user_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'spec_helper'
 
 describe User do
+  it_behaves_like "has a uuid"
```

Comment:
This was really useful during development because it helped me catch cases where individual subclasses were treating it differently for one reason or another, and also caught one case where the column had not actually been added to the class in question. Like, `Setting` was filtering out attributes in admin mode so the uuid was getting stripped before save. I never would have caught that with a single spec and we'd never catch regressions like that if they're removed, which is why I went for the shared example group. We can remove this now that we do know it's working if you want, this special case shouldn't be around long enough for us to introduce regressions once the uuid becomes the primary key, but it was worthwhile to have it there.